speech/unit_tests/test_client.py

```diff
@@ -199,11 +199,10 @@ def test_sync_recognize_with_empty_results_gax(self):
         credentials = _Credentials()
         client = self._makeOne(credentials=credentials, use_gax=True)
         client.connection = _Connection()
+        _MockGAPICSpeechAPI._results = []
```

Comment:
Why is this necessary? Can't you just create an instance of `_MockGAPICSpeechAPI`?